src/module/module-collection.js

```diff
@@ -98,11 +98,11 @@ function assertRawModule (path, rawModule) {
 }
 
 function makeAssertionMessage (path, key, type, value) {
-  let buf = `${key} should be function but "${key}.${type}"`
+  let buf = `${key} should be function but type of "${key}.${type}"`
   if (path.length > 0) {
     buf += ` in module "${path.join('.')}"`
   }
-  buf += ` is ${JSON.stringify(value)}.`
+  buf += ` is ${typeof value}.`
```

Comment:
Printing the value itself is intentional because it would be easier to aware a wrong point than printing type.